code/ui/blocks/src/blocks/Story.tsx

```diff
@@ -50,84 +86,73 @@ export const getStoryId = (props: StoryProps, context: DocsContextProps): StoryI
   return id || context.storyIdByName(name);
 };
 
+// Find the first option that isn't undefined
+function getProp<T>(...options: (T | undefined)[]) {
+  return options.find((option) => typeof option !== 'undefined');
+}
+
 export const getStoryProps = <TFramework extends Renderer>(
-  { height, inline }: StoryProps,
-  story: StoryType<TFramework>
+  props: StoryParameters,
+  story: PreparedStory<TFramework>,
+  context: DocsContextProps<TFramework>
 ): PureStoryProps => {
-  const { name: storyName, parameters = {} } = story || {};
+  const { parameters = {} } = story || {};
   const { docs = {} } = parameters;
+  const storyParameters = (docs.story || {}) as StoryParameters;
 
   if (docs.disable) {
     return null;
   }
 
-  // prefer block props, then story parameters defined by the framework-specific settings and optionally overridden by users
-  const { inlineStories = false, iframeHeight = 100 } = docs;
-  const storyIsInline = typeof inline === 'boolean' ? inline : inlineStories;
-
+  // prefer block props, then story parameters defined by the framework-specific settings
+  // and optionally overridden by users
+
+  // Deprecated parameters
+  const {
+    inlineStories,
+    iframeHeight,
+    autoplay: docsAutoplay,
+  } = docs as {
+    inlineStories?: boolean;
+    iframeHeight?: string;
+    autoplay?: boolean;
+  };
+  const inline = getProp(props.inline, storyParameters.inline, inlineStories) || false;
+
+  const height = getProp(props.height, storyParameters.height, iframeHeight) || '100px';
+  if (inline) {
+    const autoplay = getProp(props.autoplay, storyParameters.autoplay, docsAutoplay) || false;
```

Comment:
Maybe some context I'm missing, but is the `autoplay` option only added if the `inline` option is?

I don't see anywhere here where `inline` defaults to true, but `WithInteractionsAutoplayInStory` story indicates that this does indeed work, so I'm confused here.

Reply:
Update all the renderers to use `parameters.docs.story.inline`

Reply:
We discussed this -- the reason that it defaults to true is that (almost all) renderers set `parameters.docs.inlineStories` to true (e.g. https://github.com/storybookjs/storybook/blob/fe4a5960cbe3eb76626df44d384bc441e3c03d7b/code/renderers/react/src/docs/config.ts#L8)